float64 lat
float64 lon
float64 h_meter
float64 angle
